src/main.rs

```diff
@@ -20,32 +20,13 @@ fn main() -> Result<(), Box<dyn Error>> {
         process::exit(0);
     }
 
-    let is_fix = args.is_present("fix");
+    warnings.iter().for_each(|w| println!("{}", w));
 
-    if is_fix {
-        let (fixed, unfixed): (Vec<_>, Vec<_>) = warnings.iter().partition(|w| w.is_fixed);
-
-        if !fixed.is_empty() {
-            println!("Fixed warnings:");
-            for w in fixed {
-                println!("{}", w);
-            }
-        }
-
-        if !unfixed.is_empty() {
-            println!("\nUnfixed warnings:");
-            for w in unfixed {
-                println!("{}", w);
-            }
-        } else {
-            process::exit(0);
-        }
-    } else {
-        warnings.iter().for_each(|w| println!("{}", w));
-
-        if !args.is_present("quiet") {
-            print_total(warnings.len());
-        }
+    if args.is_present("fix") {
+        println!("\nAll warnings are fixed");
+        process::exit(0);
+    } else if !args.is_present("quiet") {
+        print_total(warnings.len());
```

Comment:
Should we print the total for fixed warnings too?

Reply:
I would say yes, it's always nice to see some stats when running a tool.

Reply:
I like that idea! Thank you! 🙏